devdocs/process.md

```diff
@@ -82,8 +82,10 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      doesn't always result in a good layout.
         
    * Ensure _dev green_ (i.e., all tests are passing on dev server).
+
+    >If any of the browsertests fail, use [GodMode](https://docs.google.com/) to fix them. Run the test suite again without GodMode and ensure _dev green_
```

Comment:
@wkurniawan07 How do we make sure dev changed the html file using god mode and not manually? At the moment reviewer asks the dev to confirm, but the reviewer might forget to do that. Perhaps the God mode can add a time stamp to the bottom of the file if there is any change to the file? This time stamp needs to be ignored during the comparison. Then, the reviewer should check for the time stamp when there is a change to the html file.